airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/load/data/AirbyteValue.kt

```diff
@@ -48,6 +51,24 @@ sealed interface AirbyteValue {
                         "Unrecognized value (${value.javaClass.name}: $value"
                     )
             }
+
+        fun airbyteTypeOf(value: AirbyteValue): AirbyteType =
+            when (value) {
+                // ArrayType requires the type of the object which we do not convey
```

Comment:
Should it be converted to a string to the time being?

Reply:
updated